nislmigrate/argument_handler.py

```diff
@@ -234,12 +243,17 @@ def __add_logging_flag_options(parser: ArgumentParser) -> None:
             const=logging.DEBUG,
             default=logging.WARNING)
         parser.add_argument(
-            '-v',
-            '--verbose',
-            help=VERBOSE_VERBOSITY_ARGUMENT_HELP,
+            '-s',
+            '--silent',
```

Comment:
Why no constant for this argument?

Reply:
No reason other than the fact that they aren't referenced anywhere else and I didn't want to get carried away and make even the characters constants. I went ahead and got carried away though and made the constants more consistent.